lib/zendesk_apps_tools/command.rb

```diff
@@ -299,5 +317,35 @@ def check_for_update
         say_status 'warning', 'Unable to check for new versions of zendesk_apps_tools gem', :yellow
       end
     end
+
+    def download_scaffold(app_dir)
+      tmp_download_name = 'scaffold-download-temp.zip'
+      manifest_pattern = /manifest.json$/
+      manifest_path = ''
+      scaffold_url = 'https://github.com/zendesk/app_scaffold/archive/master.zip'
+      begin
+        require 'open-uri'
+        require 'zip'
+        download = open(scaffold_url)
+        IO.copy_stream(download, tmp_download_name)
+        Zip::File.open(tmp_download_name) do |zip_file|
+          zip_file.each do |entry|
+            filename = entry.name.sub('app_scaffold-master/','')
+            if manifest_pattern.match?(filename)
+              manifest_path = filename[0..-14]
+            else
+              puts "Extracting #{filename}"
```

Comment:
maybe use `say_status` instead of `puts?`

Reply:
👏